doc/swift-target.md

```diff
@@ -23,60 +23,83 @@ $ antlr4 -Dlanguage=Swift MyGrammar.g4
 For a full list of antlr4 tool options, please visit the
 [tool documentation page](tool-options.md).
 
+
 ## Build your Swift project with ANTLR runtime
 
-The following instructions are assuming Xcode as the IDE:
+The following instructions assume Xcode as the IDE.
```

Comment:
we might also update the boot.py to reflect SPM changes in Swift4, since --gen-spm-module currently prints out an instruction on how user can use the generated local repo in their project Packages.swift.